src/modules/admin/controller/admin.controller.ts

```diff
@@ -11,6 +11,7 @@ import {
 } from '../dto/admin.dto';
 import { AdminService } from '../services/admin.service';
```

Comment:
let's make a practice to always use absolute paths i.e. @admin/dto/admin.dto.

Reply:
I agree with that. 
I'll go through all the imports to update them with module-names. Let me know if this comment makes sense to you: https://github.com/CyFy-Blockchain/auth/pull/15#discussion_r1859237249. 

If not, we can think of some other solution.